### PR TITLE
Module updates

### PR DESCRIPTION
**Pull request by chelsea-EYDS**

### CODE REVIEW

File: apps/backend/src/cash/cash.service.ts [hunk 1]
```diff
@@ -25,4 +25,13 @@ export class CashService {
       .distinct()
       .getRawMany();
   }
+
+  async createCashDeposit(data: CashDepositEntity): Promise<CashDepositEntity> {
+    try {
+      return await this.cashDepositRepo.save(this.cashDepositRepo.create(data));
+    } catch (err) {
+      this.appLogger.error(err);
+      throw err;
```
Comment (vyasworks) [inline apps/backend/src/cash/cash.service.ts:34]: do we need to rethrow?

File: apps/backend/src/lambdas/generateData.ts [hunk 2]
```diff
@@ -4,26 +4,25 @@ import { parseTDI } from './utils/parseTDI';
...
+import { POSDepositEntity } from '../pos/entities/pos-deposit.entity';
+import { TransactionEntity } from './../sales/entities/transaction.entity';
 import * as _ from 'underscore';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const handler = async (event?: any, context?: Context) => {
   const app = await NestFactory.createApplicationContext(AppModule);
   const appLogger = app.get(AppLogger);
-  const reconService = app.get(ReconciliationService);
+  const salesService = app.get(SalesService);
```
Comment (vyasworks) [inline apps/backend/src/lambdas/generateData.ts:25]: 👍

File: apps/backend/src/lambdas/generateData.ts [hunk 3]
```diff
@@ -119,28 +121,27 @@ export const handler = async (event?: any, context?: Context) => {
...
 
         if (fileType === FileTypes.TDI17) {
           const tdi17Details = parsed as TDI17Details[];
-          const tdi17Entities = tdi17Details.map(
-            (item) => new CashDepositEntity(item)
+          tdi17Details.map(
+            async (item: TDI17Details) =>
+              await cashService.createCashDeposit(new CashDepositEntity(item))
           );
           // TODO: Needs to go to it's own service / repository
```
Comment (vyasworks) [inline apps/backend/src/lambdas/generateData.ts:136]: I think we can remove the commented TOODs here

**Review by vyasworks — APPROVED** [review]

File: apps/backend/src/cash/cash.service.ts (hunk 1 above)
Comment (vyasworks) [reply apps/backend/src/cash/cash.service.ts:34]: I think we need to..

File: apps/backend/src/lambdas/generateData.ts (hunk 3 above)
Comment (chelsea-EYDS) [reply apps/backend/src/lambdas/generateData.ts:136]: Will do on the follow up PR if that is okay?

**Pull request merged**